src/shapes/Group.ts

```diff
@@ -91,7 +91,6 @@ export class Group
   /**
    * Used to allow targeting of object inside groups.
    * set to true if you want to select an object inside a group.\
-   * **REQUIRES** `subTargetCheck` set to true
```

Comment:
you can't just delete this.
You have to write on subTargetCheck that the value of subTargetCheck is overwritten by interactivity.